docs/recipes/Internationalizing-Your-Module.md

```diff
@@ -2,6 +2,64 @@
 
 # Internationalizing Your Module
 
-Recipe is forthcoming.
+## Module Locale Files
+
+[one-app-locale-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-locale-bundler) will bundle the locale files as a part of [one-app-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler). The [README](https://github.com/americanexpress/one-app-cli/blob/master/packages/one-app-locale-bundler/README.md) provides more information on the structure of locale files.
+
+## Utilizing Language Packs
+
+Each module must initialize it's own `IntlProvider`. [react-intl](https://github.com/formatjs/react-intl) version 3.x.x is compatible with One App and can be used to initialize an `IntlProvider` with a locale and language pack.
+
+To load a language pack leverage [loadLanguagePack](https://github.com/americanexpress/one-app-ducks) when initializing your module with [holocronModule](https://github.com/americanexpress/holocron).
+
+With the language pack loaded you can access the state for both the active locale and language pack and use those to create an `IntlProvider`.
+
+```jsx
+const Parent = ({ languageData, localeName }) => (
+  <IntlProvider locale={localeName} messages={languageData}>
+    <Child />
+  </IntlProvider>
+);
+
+const mapStateToProps = (state) => {
+  const localeName = state.getIn(['intl', 'activeLocale']);
+  const languagePack = state.getIn(
+    ['intl', 'languagePacks', localeName, 'module-name'],
+    fromJS({})
+  ).toJS();
+
+  return {
+    languageData: languagePack && languagePack.data ? languagePack.data : {},
+    localeName,
+  };
+};
+
+export const load = () => (dispatch) => dispatch(loadLanguagePack('module-name', { fallbackLocale: 'en-US' }));
+
+export default compose(
+  connect(mapStateToProps),
+  holocronModule({
+    name: 'module-name',
+    load,
+  })
+)(Parent);
+```
+
+With the `IntlProvider` created at the top level of the module, accessing the language pack is much simpler in child components.
+
+### Higer Order Component
+
+```jsx
+const Child = injectIntl(({ intl }) => <Fragment>{intl.messages.greeting}</Fragment>);
```

Comment:
Need to show what to `import { injectIntl } from 'react-intl'` in this guided step-by-step tutorial. Could you update all the examples how these methods are imported?

Reply:
Done